CHANGELOG.md

```diff
@@ -1,3 +1,38 @@
+## 0.32.0 (4 October 2023)
+
+ENHANCEMENTS:
+
+* Enhanced validation ([#1368](https://github.com/hashicorp/terraform-ls/pull/1368))
+  * Additional diagnostics for [selected invalid Terraform language constructs](https://github.com/hashicorp/terraform-ls/blob/v0.32.0/docs/validation.md#enhanced-validation) based on detected Terraform version and provider versions are provided.
```

Comment:
The link here isn't working yet as it targets the future tag we are yet to create as part of the release.

The benefit of using URL with tag in it is stability of the link.